ui/src/features/Editor/components/TopBar/components/ActionBar/index.tsx

```diff
@@ -94,7 +100,7 @@ const ActionBar: React.FC<Props> = ({
               alignOffset={2}>
               <DropdownMenuItem
                 className="flex justify-between gap-4"
-                onClick={handleProjectExport}>
+                onClick={() => handleProjectExport({ yDoc, project })}>
```

Comment:
Rather than passing yDoc and project in so that you need to do this kind of calling of handleProjectExport, its cleaner and more succinct I think to just raise handleProjectExport's use up, preferably where the other callback props are originating (or close to it)